kuma/static/js/syntax-prism.js

```diff
@@ -7,48 +7,57 @@
     languages.js = languages.javascript;
     languages.cpp = languages.clike;
 
+    // Style all as HTML initially
     var defaultBrush = 'html';
 
     // Treat and highlight PRE elements!
     $('article pre:not(.twopartsyntaxbox):not(.syntaxbox)').each(function() {
         var $pre = $(this);
         var klass = $.trim($pre.attr('class'));
 
-        // Split on ';' to accommodate for old line numbering
         var brush = defaultBrush;
         var lineSearch;
 
         // If the PRE has a child <code> tag, it's likely a copy/pasted, already-prism'd code samples.
         // Bail to avoid an error
-        if($pre.find('code').length) {
+        if ($pre.find('code').length) {
             return;
         }
 
         // Parse classname to look for brush
         var brushSearch = klass.match(/brush: ?(.*)/);
-        if(brushSearch && brushSearch[1]) {
-            brush = $.trim(brushSearch[1].replace(';', ' ').split(' ')[0].toLowerCase());
+        if (brushSearch && brushSearch[1]) {
+            // Split on ';' to accommodate for old line numbering
+            brush = $.trim(
+                brushSearch[1]
+                    .replace(';', ' ')
+                    .split(' ')[0]
+                    .toLowerCase()
+            );
         }
 
         if (!$pre.hasClass('no-line-numbers')) {
             // Prism upgrade requires adding a class to use line numbering
             $pre.addClass('line-numbers');
         }
 
-        // Style all as HTML initially
-        $pre.addClass('language-' + defaultBrush);
-
-        // Format PRE content for Prism highlighting
-        $pre.html('<code class="language-' + brush + '">' + $.trim($pre.html()) + '</code>');
+        // Format <pre> content for Prism highlighting
+        $pre.html(
+            '<code class="' +
+                // Don't highlight languages unknown to Prism
+                (brush in languages ? 'language-' + brush : '') +
```

Comment:
This restores pre&#x2011;https://github.com/mozilla/kuma/pull/5189 behaviour of&nbsp;unknown languages.